torch/csrc/distributed/c10d/init.cpp

```diff
@@ -633,7 +644,7 @@ Example::
           py::arg("host_name"),
           py::arg("port"),
           py::arg("world_size"),
-          py::arg("is_master"),
+          py::arg("is_master").noconvert(),
```

Comment:
https://pybind11.readthedocs.io/en/stable/advanced/functions.html#non-converting-arguments

Reply:
Great Catch! Should we add a brief comment as to why this is necessary?

Reply:
Sure, I'll add a comment, thanks!